### DISCUSSION TITLE
Cubemap Meshmapping using a .bin file issue

### DISCUSSION THREAD

**Discussion by sanikprogramist** [bonsai-rx/bonsai#1728] [Q&A]
Hi, I am trying to meshmap a cubemap environment corridor onto a dome. I am following the MeshMapping protocol from the BonVision website. i have gone through the steps to generate a .bin meshmapping file, which the protocol says "ready to use in your 3D bonsai workflow." However, the MeshMapping node doesn't seem to accept the .bin file as the correct format. What am I doing wrong?

here is an image of the Workflow section containing the meshMapping node, if that helps:
![image](https://github.com/bonsai-rx/bonsai/assets/67742957/ca74d009-6db0-4d3c-b118-b082662344e2)

**Comment by bruno-f-cruz** [comment]
Include the workflow and any assets that may help people reproduce your problem. Thanks!

**Comment by eabhorrocks** [comment]
[MeshMapping3D.zip](https://github.com/bonsai-rx/bonsai/files/14981810/MeshMapping3D.zip)

I'm not 100% sure, but I believe the current native version of BonVision meshmapping works for mapping pixels to visual angle and requires a .csv file as an input.

I've zipped a version that should take the .bin file produced by interpolator.py. I think it should be placed after RenderCubemap.
